unittest/IlmCtl/CMakeLists.txt

```diff
@@ -57,7 +56,14 @@ file(
         testVarying.ctl
         testVaryingLookup.ctl
         testVaryingReturn.ctl
-        testVSArrays.ctl
-    DESTINATION
-        ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
-)
+        testVSArrays.ctl)
+        
+if (MSVC)
+  file(COPY ${ILM_CTL_TEST_FILES} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Debug)
+  file(COPY ${ILM_CTL_TEST_FILES} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Release)
```

Comment:
Cmake config doesn't restrict MSVC to build in Release and Debug mode.
One can choose mode directly from MSVC UI. Without copying these test files the IlmCtl test will fail .This is only for testing purpose